fix eslint issues (formatting changes only)

ndelangen opened the pull request:
telescoped off https://github.com/storybookjs/storybook/pull/15308

blake-newman committed:
feat(app-vue): add support for tsx

Previous to this change vue file paths get `ts` suffix. However; if using `<script lang="tsx">` we want `tsx` to be the suffix, this change adds `tsx` as a test rule and applies `appendTsxSuffixTo`

stevensacks committed:
fix(ArgControl): unbox the value to simplify the code

stevensacks committed:
switch from case to dict for cleaner code

shilman committed:
Merge branch 'next' into pr/14394

shilman committed:
Merge branch 'next' into pr/14394

JanJakes committed:
Use semantic version constraint for prettier (^2.2.1 instead of ~2.2.1)

ndelangen committed:
fix eslint issues (formatting changes only)

ndelangen committed:
Merge branch 'fix/update-eslint-usage' into fix/update-eslint-usage-autofix

ndelangen committed:
apply more fixes to files in /addons

ndelangen committed:
Merge branch 'fix/update-eslint-usage' into fix/update-eslint-usage-autofix

ndelangen committed:
fixes

ndelangen committed:
fixes

ndelangen committed:
/addons is clear

ndelangen committed:
I was wrong, this was still needed, urgh

ndelangen committed:
fix /app

ndelangen committed:
fixed /cypress

ndelangen committed:
fix /scripts

ndelangen committed:
fixes in /examples

ndelangen committed:
fixes in /examples

ndelangen committed:
more fixes for /examples

ndelangen committed:
fix some tests

ndelangen committed:
cleanup

ndelangen committed:
fix linting in lib/ui

ndelangen committed:
fix linting in lib/source-loader

ndelangen committed:
fix linting in /lib/router

ndelangen committed:
fix linting /lib/manager-webpack*

ndelangen committed:
fix linting /lib/csf-tools

ndelangen committed:
fix linting /lib/core-*

ndelangen committed:
fix linting /lib/components

ndelangen committed:
fix linting /lib/client-*

ndelangen committed:
fix linting /lib/cli*

ndelangen committed:
fix linting lib/builder-*

ndelangen committed:
fix remaining /lib

ndelangen committed:
fix final linting issues

ndelangen committed:
fix build

ndelangen committed:
fix build

shilman commented on addons/docs/src/frameworks/react/jsxDecorator.test.tsx:
```diff
@@ -108,34 +108,32 @@ describe('renderJsx', () => {
-          I'm forwardRef!
-        </MyExoticComponent>
-      `);
+      <[object Object]>
```
something is stripping off displayName??

ndelangen replied:
Yeah, but we don't know what..

shilman commented on .eslintrc.js:
```diff
@@ -68,5 +69,12 @@ module.exports = {
+    {
+      files: ['**/*.stories.*'],
+      rules: {
+        'react/display-name': 'off',
```
Is this causing problems somehow?

ndelangen replied:
Wouldn't know how this might causes issues really.

Commit:
remove glob-base dependency

IgorSzyporyn committed:
fix: disabling eslint rule react/display-name for examples and tests

stefan-schweiger committed:
Fixed incomplete property metadata when using inheritance

ndelangen committed:
Merge branch 'next' into fix/update-eslint-usage-autofix

# Conflicts:
#	addons/a11y/src/components/Report/Info.tsx
#	addons/backgrounds/src/decorators/withBackground.ts
#	addons/docs/src/frameworks/web-components/config.js
#	addons/docs/src/frameworks/web-components/custom-elements.ts
#	lib/cli/src/versions.json
#	lib/core-client/src/preview/loadCsf.ts
#	lib/ui/src/components/sidebar/Menu.tsx

ndelangen committed:
Merge branch 'fix/update-eslint-usage' into fix/update-eslint-usage-autofix

ndelangen committed:
fix snapshots

shilman committed:
Components: Lazy-load syntax highlighter

shilman committed:
Merge pull request #15607 from storybookjs/fix/lazy-load-syntax-highlighter

Components: Lazy-load syntax highlighter

jonspalmer committed:
Upgrade server to CSF3

Commit:
Fix docs on granular viewports

jonspalmer committed:
Fix tests

apalumbo committed:
fix: trailing comma con '-s' command line params that breaks chromatic #15614

benbender committed:
[Fix] Enable react-refresh on Webpack5

Per default installation, even on `@next`, you'll end up with the following error if you enable `react-refresh` in conjunction with `webpack5`:

````
Error: NormalModuleFactory.afterResolve (CaseSensitivePathsPlugin, ReactRefreshPlugin) is no longer a waterfall hook, but a bailing hook instead. Do not return the passed object, but modify it instead. Returning false will ignore the request and results in no module created.
````

The error can be fixed by manually override the package resolution for `react-refresh` and `@pmmmwh/react-refresh-webpack-plugin` to their latest versions. To render the override obsolete, I would like to kindly ask to request to uplift both dependencies for @next.

jonspalmer committed:
Support more options for CSF autoTitle

jonspalmer committed:
startCase CSF3 autoTitles

jonspalmer committed:
use new stories config pattern

shilman committed:
Merge pull request #15615 from apalumbo/next

Trailing comma handling for "-s" command line paramenter

tdorsey committed:
docs(recipes): Fix grammatical error

graup committed:
Fix react-ts example to be runable standalone

graup committed:
Fix dependency versions

shilman committed:
Merge pull request #15620 from tdorsey/patch-1

docs(recipes): Fix grammatical error

shilman committed:
Controls: Adjust no addon-docs warning message

shilman committed:
Merge pull request #15625 from storybookjs/fix/11442-adjust-warning

Controls: Adjust no addon-docs warning message

shilman committed:
Merge pull request #15621 from graup/fix/react-ts-standalone

Examples: Fix react-ts to be runnable standalone

shilman committed:
Merge branch 'next' into pr/11936

shilman committed:
Merge pull request #11936 from blake-newman/patch-1

Vue: Add support for tsx

shilman committed:
Merge branch 'next' into pr/14394

shilman committed:
6.4.0-alpha.19 changelog

jonniebigodes committed:
Merge pull request #15613 from reubsinit/fix-viewport-docs

Fix docs on granular viewports

shilman committed:
Merge branch 'next' of github.com:storybookjs/storybook into next

shilman committed:
Merge pull request #14394 from stevensacks/fix/ArgControl-cleanup

Controls: Clean up arg unboxing and switch statements

jonniebigodes committed:
minor fixes to the list-story-reuse mdx snippet

jonniebigodes committed:
Merge pull request #15636 from storybookjs/fix_angular_snippet_writing_stories

Chore: (Docs) Minor fixes to the list-story-reuse mdx snippet

jonniebigodes committed:
adds measure and outline to the addons

jonniebigodes committed:
Merge pull request #15637 from storybookjs/update_readme_with_measure_outline

Chore: (Docs) Adds measure and outline main readme

shilman committed:
Merge branch 'next' into pr/15399

jonniebigodes committed:
Adds documentation and changes for the measure and outlien

shilman committed:
Merge pull request #15649 from storybookjs/chore_docs_add_measure_outline_docs

Chore: (Docs) Adds documentation and changes for the measure and outline addon

tmeasday committed:
Don't set arg in validateOptions if it would be `undefined`.

Fixes #15630

shilman committed:
Merge pull request #15654 from storybookjs/15630-undefined-defaults

Controls: Don't set arg in validateOptions if it would be `undefined`

shilman committed:
6.4.0-alpha.19 changelog

shilman committed:
Update root, peer deps, version.ts/json to 6.4.0-alpha.19 [ci skip]

shilman committed:
v6.4.0-alpha.19

shilman committed:
Update git head to 6.4.0-alpha.19, update yarn.lock

shilman committed:
6.4.0-alpha.19 next.json version file

shilman committed:
6.3.5 changelog

Commit:
remove obsolete dirname check

Commit:
add test cases for to-require-context function

shilman committed:
Merge pull request #15399 from lkuechler/remove-glob-base-dependency

remove glob-base dependency

shilman committed:
CSF3: Fix custom render function

shilman committed:
Merge pull request #15668 from storybookjs/fix/custom-render-function

CSF3: Fix custom render function

shilman committed:
6.4.0-alpha.20 changelog

shilman committed:
Update root, peer deps, version.ts/json to 6.4.0-alpha.20 [ci skip]

shilman committed:
v6.4.0-alpha.20

shilman committed:
Update git head to 6.4.0-alpha.20, update yarn.lock

shilman committed:
6.4.0-alpha.20 next.json version file

darleendenno committed:
fix(ts): allow ButtonLink to accept href attribute

shilman committed:
Merge pull request #15671 from storybookjs/fix/button-link-props

Allow Button to accept href attribute

vishmimoney committed:
Fix:15600 Show debug webpack output in static build

tewson committed:
Controls: fix bool control style to match value

When `true`, the toggle slider should rest on the label "True".

shilman committed:
Merge pull request #15676 from tewson/fix-boolean-control-style

Controls: Fix boolean toggle style to match underlying value

shilman committed:
Merge pull request #15674 from vishmimoney/vishmimoney/fix/debug-wepback-15600

CLI: Fix debug webpack output in static build

shilman committed:
6.4.0-alpha.21 changelog

shilman committed:
Update root, peer deps, version.ts/json to 6.4.0-alpha.21 [ci skip]

shilman committed:
v6.4.0-alpha.21

shilman committed:
Update git head to 6.4.0-alpha.21, update yarn.lock

shilman committed:
6.4.0-alpha.21 next.json version file

shilman committed:
6.3.6 changelog

shilman committed:
Merge branch 'next' into pr/15616

shilman committed:
Merge branch 'next' into pr/15586

shilman committed:
Merge pull request #15616 from benbender/patch-1

Upgrade react-refresh plugin to fix fast refresh on Webpack5

gaetanmaisse committed:
chore: upgrade to Yarn 3

gaetanmaisse committed:
chore: use same version of `markdown-to-jsx` across the monorepo

gaetanmaisse committed:
chore: markdown-to-jsx provides its own type definitions sp remote @types package

gaetanmaisse committed:
chore: simplify Yarn config of web-components example + make it works with Yarn 3

Methuselah96 committed:
Fix some transitive peer dependency warnings

Methuselah96 committed:
Update yarn.lock

shilman committed:
Merge pull request #15687 from Methuselah96/fix-transitive-peer-deps

Fix some transitive peer dependency warnings

benbender committed:
Fix markup in svelte-examples. Svelte doesn't use JSX, but plain HTML

gaetanmaisse committed:
chore: upgrade Yarn to a specific version (3.0.0 + fix on nmLinker)

For details see:
 - https://github.com/yarnpkg/berry/pull/3183
 - https://github.com/storybookjs/storybook/pull/15682#issuecomment-887320998

yaim committed:
Update Wix Style Example Link

jonniebigodes committed:
Merge pull request #15693 from yaim/patch-1

Update Wix Style Example Link

jonniebigodes committed:
adjusts a11y image and location

shilman committed:
Merge pull request #15689 from benbender/fix/svelte-not-jsx

CLI: Fix Svelte CLI template markup

shilman committed:
Merge pull request #15696 from storybookjs/adjust_a11y_image

Chore: (Docs) Adjusts a11y image and location

shilman committed:
Merge pull request #15618 from jonspalmer/start_case_auto_title

CSF3: Add startCase to auto-generated titles

jonspalmer committed:
Add more patterns for CSF3 glob processing

jonspalmer committed:
Revert example to use the new glob patterns for auto titles

shilman committed:
Merge pull request #15697 from storybookjs/csf3_glob_processing

CSF3: Add auto-titles from standard glob patterns

shilman committed:
Merge pull request #15698 from storybookjs/server_csf3

Server: Upgrade to  csf3

shilman committed:
6.4.0-alpha.22 changelog

shilman committed:
Update root, peer deps, version.ts/json to 6.4.0-alpha.22 [ci skip]

shilman committed:
v6.4.0-alpha.22

shilman committed:
Update git head to 6.4.0-alpha.22, update yarn.lock

shilman committed:
6.4.0-alpha.22 next.json version file

gaetanmaisse committed:
Merge pull request #15682 from storybookjs/upgrade-to-yarn-3

Build: Upgrade to Yarn 3

shilman committed:
Update inheritance.stories.storyshot

shilman committed:
Merge pull request #15586 from stefan-schweiger/15580-property-metadata

Angular: Fix incomplete property metadata when using inheritance

jonniebigodes committed:
adjusts the assets to prevent 404

jonniebigodes committed:
Merge pull request #15713 from storybookjs/fix_storysource_images

Chore: (Docs) Fixes the images within Storyshots addon to prevent 404

ndelangen committed:
Merge pull request #15298 from JanJakes/prettier-version-constraint

Use semantic version constraint for prettier (^2.2.1 instead of ~2.2.1)

ndelangen committed:
Merge branch 'next' into fix/update-eslint-usage-autofix

# Conflicts:
#	yarn.lock

ndelangen committed:
fix linting

shilman commented:
closing this alongside #15308
